ca.desrt.dconf-editor.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "ca.desrt.dconf-editor",
+    "base-version": "master",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "dconf-editor",
+    "rename-icon": "dconf-editor",
+    "copy-icon": true,
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--socket=wayland",
+	    "--talk-name=ca.desrt.dconf"
```

Comment:
I've not tested this yet but I'm guessing you need access to the standard dconf locations:

```
        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```